src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -19,87 +24,246 @@
 ANTISMASH_DBV2_PAGE_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/'
 ANTISMASH_DBV2_DOWNLOAD_URL = 'https://antismash-dbv2.secondarymetabolites.org/output/{}/{}'
 
-NCBI_LOOKUP_URL_NEW = 'https://www.ncbi.nlm.nih.gov/assembly/?term={}'
+NCBI_LOOKUP_URL = 'https://www.ncbi.nlm.nih.gov/assembly/?term={}'
 
 JGI_GENOME_LOOKUP_URL = 'https://img.jgi.doe.gov/cgi-bin/m/main.cgi?section=TaxonDetail&page=taxonDetail&taxon_oid={}'
 
 USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0) Gecko/20100101 Firefox/86.0'
 
+GENOME_STATUS_FILENAME = "genome_status.csv"
 
-class GenomeStatus:
 
-    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+class GenomeStatus:
+    """To represent the data status for a certain genome ID.
+
+    Attributes:
+        original_id(str): original ID for the genome. 
+        resolved_refseq_id(str): RefSeq ID generated by resolving the original genome ID. 
+        attempted(str): If the attempt of resolving the genome ID has already been done. Defaults to `False`. 
+        filename(str): Path to the file that stores the genome data. Defaults to "". 
+    """
+
+    def __init__(self,
+                 original_id: str,
+                 resolved_refseq_id: str,
+                 attempted: str = 'False',
+                 filename: str = ""):
         self.original_id = ';'.join(original_id.split(','))
```

Comment:
```suggestion
        self.original_id = original_id
```
The `original_id` can be only one id.